index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="./style.css" />
+    <title>My First Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <div id="logo-wraper">
+        <a href="#" id="logo">My Logo</a>
+      </div>
+      <nav id="app-bar">
+        <div id="hamburger-button">
+          <img src="./visuals/hamburger-icon.png" />
+        </div>
+      </nav>
+    </header>
+    <section id="headline">
+      <div id="headline-container">
+        <h1 class="title element" id="headline-title">
+          I’m Javier. Glad to see you!
+        </h1>
+        <p class="paragraph element">
+          I’m a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <h2 class="supporting-text element">LET’S CONNECT</h2>
+        <ul class="element" id="social-media-container">
+          <l1>
+            <a href="#">
+              <img class="icon" src="./visuals/twitter-icon.png" />
+            </a>
+          </l1>
+          <l1>
+            <a href="#">
+              <img class="icon" src="./visuals/linkedin-icon.png" />
+            </a>
+          </l1>
+          <l1>
+            <a href="#">
+              <img class="icon" src="./visuals/m-icon.png" />
+            </a>
+          </l1>
+          <l1>
+            <a href="#">
+              <img class="icon" src="./visuals/github-icon.png" />
+            </a>
+          </l1>
+          <l1>
+            <a href="#">
+              <img class="icon" src="./visuals/love-peace-icon.png" />
+            </a>
```

Comment:
- [ ] [OPTIONAL] Great job with this amazing webpage :clap: I feel its touch and design so well :wink However, I would suggest that you also give preference to other kinds of users who may be impaired and require special devices to access your webpage. This is called accessibility, good user experience. Adding working links in the `href` attribute of a `a` tag is important to allow users to navigate safely and easily to desired sections of your page - which in this case would be to connect or contact you. Imagine a recruiter likes your skills and would like to reach out, what happens after clicking on the links? Kindly consider getting an account for each of these and add your profile link. I think it would be awesome :muscle: 
- [ ] [OPTIONAL] On the issue of site's accessibility, adding the `alt=""` attribute of a `img` tag is also a good practice in the software industry, and not just adding it but to include a descriptive caption in-between the `"..."` ("Connect on Twitter"). This allows users to efficiently access your content in the event of these:
  - Use of screen readers by visually impaired users
  - Poor internet connectivity of users
  - Defective functionality of the server resources

From the foregoing, don't you think it's more advantageous to add descriptive alt texts to your `img` tags? :smile:

Reply:
Thanks for the tips!